lib/vtls/mbedtls.c

```diff
@@ -117,6 +117,11 @@ struct mbed_ssl_backend_data {
 #define mbedtls_strerror(a,b,c) b[0] = 0
 #endif
 
+/* PSA can be used independently of TLS 1.3 */
+#if defined(MBEDTLS_USE_PSA_CRYPTO) && MBEDTLS_VERSION_NUMBER >= 0x03060000
+#define PSA_SUPPORT
```

Comment:
To get some uniformity among these feature defines we create here, I propose the minor rename (I'll rename the 1.3 define too separately)

```suggestion
#define HAS_PSA_SUPPORT
```

Reply:
Agreed, this makes more sense. I've also seen `THREADING_SUPPORT` in the file, perhaps another candidate for renaming. Thx :-)

Reply:
yes, I fixed that one as well!